SIL.Archiving.Tests/SimpleObjectTests.cs

```diff
@@ -31,5 +31,12 @@ public void LanguageStringComparer_CorrectlyComparesLanguageStrings()
 			Assert.AreEqual(firstValue, hs.First(ls => ls.Iso3LanguageId == "eng").Value);
 			Assert.AreEqual(frenchValue, hs.First(ls => ls.Iso3LanguageId == "fra").Value);
 		}
+
+		[Test]
+		public void ToLatinOnly_MultiplePeriod_OnePeriod()
+		{
+			var result = "Langauge.wav.meta".ToLatinOnly("_", "+", ".");
+			Assert.AreEqual("Langauge+wav.meta", result);
```

Comment:
I would expect a `_` rather than a `+` here (see my comments on the `ToLatinOnly` function).